features/gesplitst-zakelijk-recht.feature

```diff
@@ -0,0 +1,429 @@
+                    }
+                ]
+            }
+        }
+        """
+
+    Scenario: opvragen van de zakelijk gerechtigden met type van gesplitst recht
+        Als zakelijk gerechtigde wordt gevraagd met "/kadastraalonroerendezaken/59020139970000/zakelijkgerechtigden?fields=identificatie,type,persoon.omschrijving&type=erfpachter"
+        Dan is het antwoord:
+        """
+        {
+            "_links": {
+                "self": {
+                    "href": "/kadastraalonroerendezaken/59020139970000/zakelijkgerechtigden?fields=identificatie,type,persoon.omschrijving&type=erfpachter"
+                },
+                "kadastraalOnroerendeZakenUitSplitsing": {
+                    "href": "/kadastraalonroerendezaken?kadastraleAanduiding=Groningen+P+1399&inclusiefKadastraalOnroerendeZakenUitSplitsing=true"
+                },
+                "kadastraalOnroerendeZakenUitSplitsingInclusiefZakelijkGerechtigden": {
+                    "href": "/kadastraalonroerendezaken?expand=zakelijkGerechtigden&kadastraleAanduiding=Groningen+P+1399&inclusiefKadastraalOnroerendeZakenUitSplitsing=true"
+                }
+            },
+            "_embedded": {},
+            "gesplitstZakelijkRecht": "erfpachter"
+        }
+        """
+
+Rule: bij een onroerende zaak met gesplitst zakelijk recht wordt het gesplitste recht getoond in gesplitstZakelijkRecht
+
+    Scenario: opvragen van een perceel met gesplitst zakelijk recht
+        Als een kadastraal onroerende zaak wordt geraadpleegd met "/kadastraalonroerendezaken/59020139970000?fields=identificatie,type,gesplitstZakelijkRecht"
+        Dan is het antwoord:
+        """
+        {
+            "identificatie": "59020139970000",
+            "domein": "NL.IMKAD.KadastraalObject",
+            "type": "perceel",
+            "gesplitstZakelijkRecht": "erfpachter",
+            "_links": {
+                "self": {
+                    "href": "/kadastraalonroerendezaken/59020139970000?fields=identificatie,type,gesplitstZakelijkRecht"
+                }
+            }
+        }
+        """
+
+    Scenario: opvragen van een appartementsrecht uit gesplitst zakelijk recht
+        Als een kadastraal onroerende zaak wordt geraadpleegd met "/kadastraalonroerendezaken/59020140010001?fields=identificatie,type,gesplitstZakelijkRecht"
+        Dan is het antwoord:
+        """
+        {
+            "identificatie": "59020140010001",
+            "domein": "NL.IMKAD.KadastraalObject",
+            "type": "appartementsrecht",
+            "_links": {
+                "self": {
+                    "href": "/kadastraalonroerendezaken/59020140010001?fields=identificatie,type,gesplitstZakelijkRecht"
+                }
+            }
+        }
+        """
+
+Rule: De onroerende zaken waarin het zakelijk recht van een onroerende zaak is gesplitst kunnen worden opgevraagd door gebruik van de parameter inclusiefKadastraalOnroerendeZakenUitSplitsing in combinatie met kadastraleAanduiding
```

Comment:
Wat is het verschil tussen deze rule en de rule: De onroerende zaken waarin het zakelijk recht van een onroerende zaak is gesplitst kunnen worden opgevraagd door gebruik van de parameter inclusiefKadastraalOnroerendeZakenUitSplitsing in combinatie met kadastraleAanduiding?
Als je een KOZ opvraagt die is gesplitst en je zet de inclusiefKadastraalOnroerendeZakenUitSplitsing property dan is het de bedoeling dat je in het antwoord ook de KOZ-en krijgt die zijn ontstaan uit splitsing?

Reply:
@MelvLee hoe bedoel je dit? Heb je deze rule nog ergens gevonden (is die dubbel opgenomen)? Ik heb dat niet gevonden

Reply:
Ik bedoel de volgende rule: Wanneer een appartementsrecht is (onder)gesplitst wordt bij opvragen van het grondperceel inclusiefKadastraalOnroerendeZakenUitSplitsing zowel het gesplitste appartementsrecht als de appartementsrechten die daaruit zijn ontstaan geleverd (regel 302). Mijn vraag bij verkeerde regel geplaatst.

Reply:
de eerste rule (regel 150) beschrijft dat de appartementsrechten die uit het perceel zijn opgesplitst worden teruggegeven
de tweede (regel 302) beschrijft dat ook appartementsrechten die zijn ondergesplitst worden meegegeven

dit is dus een verdere uitwerking voor een bijzondere situatie